etc/kayobe/inventory/group_vars/cis-hardening/cis

```diff
@@ -74,38 +74,39 @@ rhel9cis_set_boot_pass: false
 rhel9cis_rule_5_6_1_1: false
 
 ##############################################################################
-# Ubuntu Jammy CIS Hardening Configuration
+# Ubuntu Noble CIS Hardening Configuration
+# FIXME: These settings are untested, they are just carried over from Jammy
 
-# Ubuntu 22 CIS configuration
+# Ubuntu 24 CIS configuration
 # Disable changing routing rules
-ubtu22cis_is_router: true
+ubtu24cis_is_router: true
 
 # Set Chrony as the time sync tool
-ubtu22cis_time_sync_tool: "chrony"
+ubtu24cis_time_sync_tool: "chrony"
 
 # Disable CIS from configuring the firewall
-ubtu22cis_firewall_package: "none"
+ubtu24cis_firewall_package: "none"
 
 # Stop CIS from installing Network Manager
-ubtu22cis_install_network_manager: false
+ubtu24cis_install_network_manager: false
 
 # Set syslog service to journald
-ubtu22cis_syslog_service: journald
+ubtu24cis_syslog_service: journald
 
 # Squashfs is compiled into the kernel
-ubtu22cis_rule_1_1_1_2: false
+ubtu24cis_rule_1_1_1_2: false
 
 # This updates the system. Let's do this explicitly.
-ubtu22cis_rule_1_9: false
+ubtu24cis_rule_1_9: false
 
 # Do not change Chrony Time servers
-ubtu22cis_rule_2_1_2_1: false
+ubtu24cis_rule_2_1_2_1: false
```

Comment:
Just fyi, these will need to be checked and fixed. `ubtu24cis_rule_2_1_2_1` doesn't exist for example https://github.com/ansible-lockdown/UBUNTU24-CIS-Audit/blob/devel/vars/CIS.yml#L140